.github/workflows/endpoint-test.yml

```diff
@@ -41,5 +45,9 @@ jobs:
 
       - name: Run System Tests
         env:
-            TEST_ENVIRONMENT: ${{ matrix.env }}
-        run: pytest -sv test/system-test/test_${{ inputs.endpoint }}.py
+          TEST_ENVIRONMENT: ccf/${{ matrix.config.env }}
+        run: |
+          source ./scripts/jwt_issuer/up.sh
+          pytest -sv \
+            test/system-test/test_${{ inputs.endpoint }}.py \
+            -n ${{ matrix.config.threads }}
```

Comment:
cool. Can we use -n on our other CI's?

Reply:
If by other CI's you mean using the `sandbox_local` environment, no because it's currently got hardcoded endpoint ports so if you tried to run multiple concurrently they would clash.